src/app/team/page.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import { Outfit } from 'next/font/google';
```

Comment:
Remove this and use `font-outfit` utility.

Reply:
done